shell/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -212,11 +212,14 @@ AVMediaType ParseMediaType(const std::string& media_type) {
                     NSDictionaryToDictionaryValue(notification.userInfo);
                 if (user_info) {
                   copied_callback.Run(
-                      base::SysNSStringToUTF8(notification.name), *user_info);
+                      base::SysNSStringToUTF8(notification.name),
+                      *user_info,
+                      base::SysNSStringToUTF8(notification.object));
```

Comment:
`notification.object` has type `id`, which doesn't necessarily point to an `NSString`. `base::SysNSStringToUTF8` will crash if you pass it an object that isn't an `NSString`. I'm somewhat surprised that this compiles, but in any case we should definitely check that the object is actually a string, e.g.

```objc
if ([notification.object isKindOfClass:[NSString class]]) {
  // ...
}
```

Reply:
Thanks for the spotting this @nornagon, and for including sample code in your feedback. Made this a really easy fix for me!